src/server/webpack-config.js

```diff
@@ -124,6 +127,74 @@ export default (options = {}) => {
     ],
   };
 
+  // Configure CSS loaders
+  const { loaders } = config.module;
+  const { cssModules } = options;
+  const simpleCssLoader = { test: /\.css$/, loader: 'style!css' };
+  if (cssModules) {
+    let simpleLoaderAdded = false;
+    cssModules.forEach(test => {
+      loaders.push({
+        test,
+        loader: 'style!css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', // eslint-disable-line max-len
+        exclude: /node_modules/,
```

Comment:
@philcockfield can you make sure this line doesn't break the ui-harness when it's used in other projects - i.e. css modules are still loaded correctly from npm packages... I have a sneaking suspicion they won't be